microsoft-edge/extensions-chromium/publish/verify-microsoft-edge-program.md

```diff
@@ -0,0 +1,90 @@
+| Business | Business verification confirms that the enrolling company is a legitimate business entity and at the stated address. | Submit one of the following documents for business verification: <br> * Formation documents, such as articles of incorporation, partnership deed. <br> * Franchise or agency appointment letters. <br> * Government issued letter, license, business registration, or tax registration certificate. <br> * Lease or tenancy documents. <br> * Letter or statement from a financial institution or a utility company. <br> * Record on a Government registry website.The site/link must be displayed. <br> * Stock exchange filings or tax filing records. <br> * External company databases, such as Dun & Bradstreet (DUNS ID) or state registry. <br> * Receipt from the company's home country, municipality. <br> <br> **Note:** Confirm that the company name and address in your [Legal business profile](https://partner.microsoft.com/dashboard/account/v3/organization/legalinfo#developer) have no spelling errors or abbreviations. They must match your formal company business registration records exactly. If appropriate, select the match found in external data sources: external company databases, such as Dun & Bradstreet (DUNS ID) or state registry. <br> <br> Microsoft uses this documentation to verify that the company is authorized to do business under that name, and that it is located at the address provided. For further assistance, go to your profile page to upload additional proofs through the [interactive vetting experience](#checking-your-verification-status) |
+
+
+<!-- ====================================================================== -->
+## Checking your verification status
+
+You can check verification status at Partner Center in **[Account Settings | Legal Info](https://partner.microsoft.com/dashboard/account/v3/organization/legalinfo#developer)**, where you can see:
+
+1. One or more tabs such as **Partner**, **Developer**, or **Reseller**, depending on the programs in which your organization is enrolled.
+1. Your verification status: **Pending**, **Accepted**, or **Rejected**, with a status icon.
+1. The **Legal business profile** progress bar with an information icon you can select to get more information.
```

Comment:
Suggesting a change in this whole section given that the UI has changed:

You can check verification status at Partner Center in **[Account Settings | Legal Info](https://partner.microsoft.com/dashboard/account/v3/organization/legalinfo#developer)**, where you can see:

1. One or more tabs such as **Partner**, **Developer**, or **Reseller**, depending on the programs in which your organization is enrolled. If you have registered for the Microsoft Edge program you will see the Developer tab. 
1. On the developer tab, you will see the **Account details** progress bar showing the various stages of verification and the pending stages. 
1. Your verification status: **Pending**, **Authorized/Accepted**, or **Action Required/Rejected**, with a status icon.
1. A "Fix Now" button.

Reply:
@[ashishnirvana1 Is the "The **Legal business profile** progress bar with an information icon you can select to get more information." line still needed?